### ISSUE TITLE
Timer op rechten van leden

### ISSUE THREAD

**Issue by Dyxl3ct1c** [csrdelft/csrdelft.nl#399]
Op deze manier wordt voorkomen dat leden langer toegang hebben tot bepaalde rechten dan nodig.

**Comment by mathcals** [comment]
Ja, misschien ook handig om het systeem überhaupt zo flexibel te maken dat iemand meerdere rollen heeft. (Zodat we geen gekke rollen als R_VLIEGER) nodig hebben
